user_sync/app.py

```diff
@@ -299,6 +299,17 @@ def create_config_loader_options(args):
     
     return config_options
 
+def log_parameters(args):
+    '''
+    Log the invocation parameters to make it easier to diagnose problem when customers
+    :param args: namespace
+    :return: None
+    '''
+    logger.info('----- Invocation Parameters -----')
+    for parameter_name, parameter_value in args.__dict__.iteritems():
+        logger.info('  %s: %s', parameter_name, parameter_value)
+    logger.info('---------------------------------')
```

Comment:
So this looks like it logs our internal (code-level) parameter names and values, which is good, but it doesn't get at the essence of the ask from a customer perspective, which is to actually put the command line that the customer gave us in the log.  Can you please *add* to what you have here additional code which essentially logs the sys.argv command line?